target-app/Home/Views/HomeViewController.swift

```diff
@@ -24,22 +45,38 @@ class HomeViewController: UIViewController {
         setupMapConstraints()
     }
     
+    override func viewDidAppear(_ animated: Bool) {
+        LocationManager.shared.requestLocationAuthorization()
+        
+        if CLLocationManager.locationServicesEnabled() {
+            locationManager.delegate = self
+            locationManager.desiredAccuracy = kCLLocationAccuracyBest
+            locationManager.startUpdatingLocation()
+        }
+        
+        mapView.delegate = self
+        
+        if let coor = mapView.userLocation.location?.coordinate{
```

Comment:
pls avoid the use of acronyms and abbreviations. 
```suggestion
        if let coordinates = mapView.userLocation.location?.coordinate {
```